internal/objects/validator_test.go

```diff
@@ -210,7 +210,63 @@ func TestUserChanges(t *testing.T) {
 				},
 			},
 		},
-	}, {
+	},{
+		name: "Deny metadata changes to HNC annotations in propagated objects",
```

Comment:
You don't need the word "metadata" here, an annotation is more specific anyway. Ditto for the other test.